src/persistentSubscription/utils/persistentSubscriptionSettings.ts

```diff
@@ -80,7 +80,7 @@ interface PersistentSubscriptionSettingsGeneric {
 export interface PersistentSubscriptionSettings
   extends PersistentSubscriptionSettingsGeneric {
   /**
-   * The exclusive position in the stream or transaction file the subscription should start from.
+   * The inclusive position in the stream or transaction file the subscription should start from.
```

Comment:
How about something like:

```suggestion
   * Where to start the subscription from. This can be from the start of the stream, from the end of the stream at the time of creation, or from an inclusive position in the stream.
```

Reply:
@ylorph

Reply:
:+1: